CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 
 * Removed deprecated `InstructionsBannerViewDelegate.didDragInstructionsBanner(_:)` method. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * Removed `NavigationViewController.origin` property, which was not used. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
+* Added `PredictiveCacheOptions` to allow enabling and configuring a Predictive Caching mechanism which pre-fetches tiles which may become needed during navigation if app goes offline at some point. This can be configure either by updating `NavigationOptions` passed to `NavigationViewController` or by manually calling `NavigationMapView.setupPredictiveCaching(_:)` ([#2830](https://github.com/mapbox/mapbox-navigation-ios/pull/2830))
```

Comment:
Now that I’m looking closely at this changelog entry, it makes me wonder if we should’ve called the feature the “proactive cache” for consistency with the longstanding feature known as proactive rerouting. But it’s a bit too late for that change of terminology, I suppose.

```suggestion
* A new predictive cache proactively fetches tiles which may become necessary if the device loses its Internet connection at some point during passive or active turn-by-turn navigation. Pass a `PredictiveCacheOptions` instance into the `NavigationOptions(styles:navigationService:voiceController:topBanner:bottomBanner:predictiveCacheOptions:)` initializer as you configure a `NavigationViewController`, or manually call `NavigationMapView.enablePredictiveCaching(options:)`. ([#2830](https://github.com/mapbox/mapbox-navigation-ios/pull/2830))
```